examples/device-a.py

```diff
@@ -0,0 +1,13 @@
+from robocluster import Device
+
+device = Device('device-a', '224.0.0.64:32464')
+
+@device.on('device-b/hello')
+def hello(event, data):
+    print(event, data)
+
+@device.on('device-b/every')
+def every(event, data):
+    print(event, data)
+
+device.run_forever()
```

Comment:
`run_forever()` blocks right? What if we have multiple devices that need to be run? By default they would use the same event loop so I imagine starting one would start all of them, but it doesn't seem that explicit.

Reply:
We can look into this. We should probably make a new event loop for each device to make stopping it easier.

Reply:
I just briefly looked into using multiple event loops in asyncio, and we may have to also wrap the call to `run_forever` in a new thread, as it blocks regardless of what event loop its for. So `device.run_forever()` would start a new thread to call the asyncio `run_forever()` and return, allowing you to run multiple devices from the same script. I feel there may a better way though.

Reply:
I'll see what I can do about that today. I remember there being something weird you had to do to make an event loop in a background thread.